### PR TITLE
Create note UI functionality

### PR DESCRIPTION
**Pull request by dikshadhote**
Preview : https://deploy-preview-5--glittering-lily-ade5dc.netlify.app/home
Following features have been added
1. Add label
image
2.. Add color 
image

3. Add priority 
image

### CODE REVIEW

File: src/Context/notes-context.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import { useReducer, createContext, useContext } from "react";
+
+const NoteContext = createContext();
+
+const useNote = () => useContext(NoteContext);
+
+const noteReducer = (noteState, action) => {
+  console.log(action);
```
Comment (rahulmistry751) [inline src/Context/notes-context.js]: remove console.log

File: src/components/Home/Home.js [hunk 2]
```diff
@@ -1,46 +1,261 @@
...
+import { useNote } from "../../Context/notes-context";
 
 export default function Home() {
+  const { noteState, noteDispatch } = useNote();
+  const isPin = noteState.isPin;
+  const [label, setLabel] = useState("");
+  const [showLabel, setShowLabel] = useState(false);
+  const [showPallete, setShowPallete] = useState(false);
+  const [showPriority, setShowPriority] = useState(false);
+  console.log(noteState.priority);
```
Comment (rahulmistry751) [inline src/components/Home/Home.js]: remove console log

**Review by rahulmistry751 — APPROVED** [review]

File: src/Context/notes-context.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  //   sort out unique label
```
Comment (sushil-mod) [inline src/Context/notes-context.js:8]: this is correct . just a suggestion you can destructure action into payload and type beforehand .

**Review by sushil-mod — APPROVED** [review]
just a suggestion . reast code is good . UI looks nice good work

**Pull request merged**